Source/FieldSolver/WarpXPushFieldsHybridPIC.cpp

```diff
@@ -63,7 +64,12 @@ void WarpX::HybridPICEvolveFields ()
     // Perform charge deposition in component 0 of rho_fp at t_{n+1}.
     mypc->DepositCharge(m_fields.get_mr_levels(FieldType::rho_fp, finest_level), 0._rt);
     // Perform current deposition at t_{n+1/2}.
-    mypc->DepositCurrent(m_fields.get_mr_levels_alldirs(FieldType::current_fp, finest_level), dt[0], -0.5_rt * dt[0]);
+    mypc->DepositCurrent(m_fields.get_mr_levels_alldirs(FieldType::current_fp, finest_level),
+        dt[0], -0.5_rt * dt[0]);
+
+    // TODO: Perhaps add flag here for when using temperature accumulation in Hybrid
```

Comment:
Please do add the flag.
I realize `DepositTemperatures()` will return without really doing anything if all species have `pc->getTemperatureDepositionFlag() == false`, but since the temperature can be output as a diagnostic this could lead to inefficiency. If someone does not want the heat equation coupled to the Ohm solver but would like to have the individual temperature components as a diagnostic, this would result in lots of unnecessary depositions.

Reply:
This is intended to be used as a tightly coupled deposition loop for use with the electron energy equation. This can easily be adapted to run similarly to the NGP Total temperature at diagnostic intervals, but it wasn't really the purpose of the PR. I can change a few things around to do this, but I was hoping leaving the existing functionality would be fine for now. I can get back to this a bit later to clean it up for diagnostic use, but it may be a little bit before I can clean up and test that use case.